service/matching/physical_task_queue_manager.go

```diff
@@ -82,41 +79,6 @@ type (
 		forwardInfo *taskqueuespb.TaskForwardInfo
 	}
 
-	physicalTaskQueueManager interface {
```

Comment:
moving all this code around is fine, but please split it into a different PR, it has nothing to do with force-loading

Reply:
Okay! give me a bit and I'll have it split.

Reply:
I've pulled out the refactoring changes that were focused on patterns or readability.